torch/fx/symbolic_trace.py

```diff
@@ -52,7 +52,7 @@ class Tracer(TracerBase):
     """
     ``Tracer`` is the class that implements the symbolic tracing functionality
     of ``torch.fx.symbolic_trace``. A call to ``symbolic_trace(m)`` is equivalent
-    to ``Tracer().trace(m)``.
+    to ``Tracer().trace(m).graph``.
```

Comment:
`Tracer().trace(m)` returns a `Graph`, not GraphModule`. I actually think this docstring should say something like `symbolic_trace(m) is equivalent to GraphModule(m, Tracer().trace(m))`